image/image_test.go

```diff
@@ -10,11 +10,25 @@ import (
 	v1 "k8s.io/api/core/v1"
 )
 
-func generateContainerStatus(name, image, imageID string) v1.ContainerStatus {
-	cs := v1.ContainerStatus{
-		Name:    name,
-		Image:   image,
-		ImageID: imageID,
+func generatePod(name, image, imageID string) v1.Pod {
+	cs := v1.Pod{
```

Comment:
super nit, but since this isn't a `containerstatus` anymore renaming the var to something like `p`

Reply:
or even simpler just return the struct w/o a named var 🤷

Reply:
good catch, removed